remove tests that depend on gix_testtools as that isn't packaged in Debian

Index: gix-actor/src/signature/decode.rs
===================================================================
--- gix-actor.orig/src/signature/decode.rs
+++ gix-actor/src/signature/decode.rs
@@ -155,7 +155,7 @@ mod tests {
             );
         }
 
-        #[test]
+        /* #[test]
         fn tz_plus() {
             assert_eq!(
                 decode
@@ -175,7 +175,7 @@ mod tests {
                     .1,
                 signature("Sebastian Thiel", "byronimo@gmail.com", 1528473343, Sign::Minus, 0)
             );
-        }
+        } */
 
         #[test]
         fn negative_offset_double_dash() {
